packages/truffle/test/scenarios/commands/help.js

```diff
@@ -42,5 +42,29 @@ describe("truffle help [ @standalone ]", function () {
       );
       assert(output.includes("--environment"));
     });
+
+    it("displays help for given command when `--help` is in the command line", async function () {
+      await CommandRunner.run("compile --help", config);
+      const output = logger.contents();
+      assert(output.includes("Description:  Compile contract source files"));
+    }).timeout(20000);
+
+    it("displays help for the `help` command", async function () {
+      await CommandRunner.run("help help", config);
+      const output = logger.contents();
+      assert(
+        output.includes(
+          " Description:  List all commands or provide information about a specific command"
+        )
+      );
+    }).timeout(20000);
+
+    it("displays help for `truffle <cmd> --help <subCommand>` is used", async function () {
+      await CommandRunner.run("db --help serve", config);
+      const output = logger.contents();
+      assert(
+        output.includes("Description:  Start Truffle's GraphQL UI playground")
```

Comment:
I don't think this is what we want. In the case that we have `--help` in the command, we shouldn't see `serve` start up, right? I thought it was decided that in the case that we find the `--help` flag, we would display the help for that command. Am I misremembering?